### ISSUE TITLE
add inputs executionRoleArn and taskRoleArn

### ISSUE THREAD

**Issue by hblab-ngocnd** [aws-actions/amazon-ecs-render-task-definition#183]
for security reason, want add input  executionRoleArn and taskRoleArn for task definition

**Comment by bert2002** [comment]
You can simply do this by adding it to the `task-definition` . E.g.
```
  "TaskDefinition": {
    "executionRoleArn": "arn:aws:iam::<id>:role/<name>",
    "containerDefinitions": [
      {
...
```

**Comment by hblab-ngocnd** [comment]
@bert2002 
But that not good for security because, that show `Arn Role` for viewer. 
It is better If input has options for set role from secret key by input

https://github.com/aws-actions/amazon-ecs-render-task-definition/blob/74bfbc88d5d3a3c237d9b643ed1b18546483b89e/action.yml#L6-L18

**Comment by dangeReis** [comment]
I agree, hardcoding the parameters in the task-definition is not useful.  Can we prioritize this feature request?
